build.vsh

```diff
@@ -33,8 +33,21 @@ if vls_git_hash.exit_code != 0 {
 }
 os.setenv('VLS_BUILD_COMMIT', vls_git_hash.output.trim_space(), true)
 
-use_libbacktrace_flag := if cc == 'msvc' { '' } else { '-d use_libbacktrace' }
-cmd := 'v -g -gc boehm $use_libbacktrace_flag -cc $cc cmd/vls -o $full_vls_exec_path'
+mut buffer := ['v', 'cmd/vls']
+buffer << '-g'
+buffer << ['-o', full_vls_exec_path]
+buffer << ['-gc', 'boehm']
+buffer << ['-cc', cc]
+if cc != 'msvc' {
+	buffer << ['-d', 'use_libbacktrace']
+}
+
+index_extra := os.args.index('--')
+if index_extra > 0 {
+	buffer << os.args[index_extra + 1..]
```

Comment:
That can panic, if someone uses `--` as a last argument, without anything after it.

Reply:
`a[a.len..]` instead of panic, returns empty a slice now, so is this just fine?